reporter/html.js

```diff
@@ -327,44 +353,106 @@ function toolbarLooseFilter() {
 	return filter;
 }
 
-function toolbarModuleFilterHtml() {
+function moduleDropDownHtml () {
 	var i,
-		moduleFilterHtml = "";
+		dropDownHtml = "";
 
-	if ( !modulesList.length ) {
+	if ( !config.modules.length ) {
 		return false;
 	}
 
-	moduleFilterHtml += "<label for='qunit-modulefilter'>Module: </label>" +
-		"<select id='qunit-modulefilter' name='modulefilter'><option value='' " +
-		( QUnit.urlParams.module === undefined ? "selected='selected'" : "" ) +
-		">< All Modules ></option>";
-
-	for ( i = 0; i < modulesList.length; i++ ) {
-		moduleFilterHtml += "<option value='" +
-			escapeText( encodeURIComponent( modulesList[ i ] ) ) + "' " +
-			( QUnit.urlParams.module === modulesList[ i ] ? "selected='selected'" : "" ) +
-			">" + escapeText( modulesList[ i ] ) + "</option>";
+	for ( i = 0; i < config.modules.length; i++ ) {
+		if ( config.modules[ i ].name !== "" ) {
+			dropDownHtml += "<li><input id='" + config.modules[ i ].moduleId +
```

Comment:
That's also possible if the input is inside the label, no `id` needed. But if you do keep id, it should be prefixed with something like "module-".